libraries/src/MVC/Model/BaseDatabaseModel.php

```diff
@@ -330,7 +330,8 @@ protected function _getListCount($query)
 			&& $query->group === null
 			&& $query->union === null
 			&& $query->unionAll === null
-			&& $query->having === null)
+			&& $query->having === null
+			&& stripos((string) $query, 'distinct') === false)
```

Comment:
```suggestion
			&& stripos((string) strtolower($query), 'distinct') === false)
```

hmm should't we do a strtolower($query) here too as i could also write that key word in all upper case.

Reply:
stripos is already case insensitive no need to change this.